GeoidHeights/GeoidHeights.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
+    <TargetFramework>netstandard1.1</TargetFramework>
     <PackageId>GeoidHeightsDotNet</PackageId>
     <PackageVersion>1.0.2</PackageVersion>

```

Comment:
I didn't change version, but I did a test run of dotnet pack .\GeoidHeights\GeoidHeights.csproj